tests/e2e/vLLM/configs/kv_cache_gptq_tinyllama.yaml

```diff
@@ -0,0 +1,7 @@
+cadence: "nightly"
+test_type: "regression"
+model: TinyLlama/TinyLlama-1.1B-Chat-v1.0
```

Comment:
Other models/size and recipes are welcome. 
currently targeting small model and fused layer model